api/src/models/AudioFile.ts

```diff
@@ -14,14 +13,30 @@ class AudioFile {
         this.dir = dir;
     }
 
+    ffmpeg(): string {
```

Comment:
Is there value in having this ffmpeg method at all? Since this is only used in one place, maybe just moving `const ffmpeg_path = Config.getInstance().ffmpegPath;` down into the derivative method would be just as good.

If you do want to keep this (which I wouldn't object to -- just posing a question), I would suggest making this a getter, since it doesn't really need to be a method, and then you can take some parentheses off down below.